packages/react-sdk/tsconfig.json

```
@@ -1,12 +1,7 @@
 {
   // Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
   "extends": "../../node_modules/@webstudio-is/tsconfig/base.json",
-  "include": [
-    "./src/**/*.ts",
-    "./src/**/*.tsx",
-    "./src/**/*.json",
-    "./@types/**/*.d.ts"
-  ],
+  "include": ["./src", "./src/**/*.json", "./@types/**/*.d.ts"],
```

can't we simplify this one to something like this?
```suggestion
  "include": ["./src", "./@types"],
```

---

Looks like json is not part of default extensions. I get errors.